code/processes/segmentedtickerplant.q

```diff
@@ -0,0 +1,59 @@
+// Segmented TP process
+// Contains all TP functionality with additional flexibility
+// Configurable logging and subscriptions
+// Default settings create single TP log per table and rolls logs hourly
```

Comment:
I don't believe these defaults are set anywhere? I had to set

- .stplg.multilog
- .stplg.batchmode
- .stplg.errmode 

in appconfig/settings to get it to work.